backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/utils/VdsCpuUnitPinningHelper.java

```diff
@@ -86,6 +95,7 @@ private void previewPinOfPendingExclusiveCpus(List<VdsCpuUnit> cpuTopology, Map<
         }
 
         previewPinOfPendingExclusiveCpus(cpuTopology, vmToPendingPinnings, vm.getCpuPinningPolicy());
+        filterBadNumaMemorySockets(cpuTopology, vm, hostId);
```

Comment:
same here

Reply:
as we talked because we are using `getMaxFreedSocket` we may end up always with the wrong pSocket (it's based on CPUs only). filtering will make sure every pSocket we choose will be ok.